form/form_collections.rst

```diff
@@ -340,6 +340,31 @@ you'll replace with a unique, incrementing number (e.g. ``task[tags][3][name]``)
         // Add the new form at the end of the list
         $collectionHolder.append($newFormLi)
     }
+    
+Code without jQuery library
+
+.. code-block:: javascript
+    const newItem = (e) => {
+      const collectionHolder = document.querySelector(`.${e.currentTarget.dataset.collectionHolderClass}`);
```

Comment:
The JS string interpolation is nice ... but given that we're just prefixing a `.` char, and given that the JS syntax uses `${}`, which is always confusing to PHP developers because `$` is strongly associated to PHP ... maybe we could remove the interpolation and just do a normal string concatenation.

Reply:
I change change string interpolation to simple concatenation